packages/perseus/src/types.ts

```diff
@@ -155,26 +158,10 @@ export type ChangeHandler = (
         answerArea?: PerseusAnswerArea | null;
         editorMode?: EditorMode;
         jsonMode?: boolean;
-        // perseus-all-package/widgets/unit.jsx
-        value?: any;
         // widgets/radio/widget.jsx
         choiceStates?: ReadonlyArray<ChoiceState>;
-        // widgets/numeric-input.jsx
-        currentValue?: string;
-        // perseus-all-package/widgets/dropdown.jsx
-        selected?: number;
-        // perseus-all-package/widgets/grapher.jsx
-        plot?: any;
-        // Interactive Graph callback (see legacy: interactive-graph.tsx)
-        graph?: PerseusGraphType;
```

Comment:
Are all the deletions here removing stuff that was from user input?

Reply:
Yep! That's right